spec/controllers/users/additional_mfa_required_controller_spec.rb

```diff
@@ -1,21 +1,28 @@
 require 'rails_helper'
 
 describe Users::AdditionalMfaRequiredController do
+  render_views
```

Comment:
`render_views` can slow down tests a bunch, is it possible to add this only in certain contexts or certain specs? ex the ones that check for redirects?

Reply:
Yep!